README.md

```diff
@@ -44,12 +44,51 @@ Enter your DSN in the NuxtJS config file. Additional config settings can be foun
 
 In a Vue component, `Sentry` is available as `this.$sentry`, so we can call functions like
 
-```
+``` js
 this.$sentry.captureException(new Error('example'))
 ```
 
 where `this` is a Vue instance.
 
+### Usage in the `fetch` and `asyncData` methods
```

Comment:
```suggestion
### Usage in `fetch`, `asyncData`, `plugins`, `middleware`, `modules`, and `nuxtServerInit`

`$sentry` object is accessible through the `app` object contained within the [NuxtContext](https://nuxtjs.org/api/context) object. Some examples:

``` js
async fetch ({ app, params, store }) {
  try {
    let { data } = await axios.get(`https://my-api/posts/meta/${params.id}`)
    store.commit('setMeta', data)
  } catch (error) {
    app.$sentry.captureException(error)
  }
}
async asyncData ({ app, params }) {
  try {
    let { data } = await axios.get(`https://my-api/posts/${params.id}`)
    return { title: data.title }
  } catch (error) {
    app.$sentry.captureException(error)
  }
}
```

```js
async nuxtServerInit({ commit }, context) {
  try {
    let { data } = await axios.get(`https://my-api/timestamp`)
    commit('setTimeStamp', data)
  } catch (error) {
    context.app.$sentry.captureException(error)
  }
}
```

Reply:
can't get the formatting quite right on this one but you should be able to correct it.